app/admin/RTMediaAdmin.php

```diff
@@ -1533,6 +1542,84 @@ public function linkback() {
 			}
 		}
 
+		/**
+		 * Export rtMedia Settings
+		 *
+		 * @access public
+		 */
+		public function export_settings() {
+
+			$rtmedia_option = get_option( 'rtmedia-options' );
+
+			if ( is_array( $rtmedia_option ) ) {
+				$rtmedia_option['rtm_key'] = md5( 'rtmedia-options' );
+			}
+
+			wp_send_json( $rtmedia_option );
+		}
+
+		/**
+		 * Import rtMedia Settings
+		 *
+		 * @access public
+		 *
+		 * @param string $file_path path to json file to be imported.
+		 */
+		public function import_settings( $file_path = '' ) {
+
+			$response = array();
+
+			if ( ! empty( $file_path ) ) {
+
+				$settings_data_json = false;
+				if ( file_exists( $file_path ) ) {
+					ob_start();
+					include $file_path; // @codingStandardsIgnoreLine
+					$settings_data_json = ob_get_clean();
+					wp_delete_file( $file_path );
+				}
+
+				if ( false !== $settings_data_json || ! empty( $settings_data_json ) ) {
+					$settings_data = json_decode( $settings_data_json, true );
+					if ( ! is_array( $settings_data ) || ! empty( $settings_data['rtm_key'] ) ) {
```

Comment:
Avoid large `if` block. If  `$settings_data ` is empty then return data with an error message.